ci/setup_tpm12_simulator.sh

```diff
@@ -63,8 +63,46 @@ fetch_simulator () {
 build_simulator () {
   cp -v "${SIMULATOR_SRC}/tpm/makefile-tpm" "${SIMULATOR_SRC}/tpm/Makefile"
   cd "${SIMULATOR_SRC}/tpm" && make -j4
+  cd "${SIMULATOR_SRC}/libtpm" && ./autogen
+  cd "${SIMULATOR_SRC}/libtpm" && ./configure
+  cd "${SIMULATOR_SRC}/libtpm" && make -j4
+}
+
+run_simulator () {
+  mkdir -pv "${BUILD_BASE}/NVRAM"
+  export TPM_PORT='6545'
+  export TPM_PATH="${BUILD_BASE}/NVRAM"
+  export TPM_SERVER_NAME='localhost'
+  export TPM_SERVER_PORT='6545'
+  export TCSD_TCP_DEVICE_PORT='6545'
+  ${SIMULATOR_SRC}/tpm/tpm_server &
+  SIM_PID=$!
+  echo "${SIM_PID}" > "${BUILD_BASE}/sim_pid"
+  disown
+  sleep 2
+}
+
+setup_tpm () {
+  echo "Starting the TPM..."
+  ${SIMULATOR_SRC}/libtpm/utils/tpmbios -v
+  echo "Creating a fake EK..."
+  ${SIMULATOR_SRC}/libtpm/utils/createek || true
```

Comment:
Why || true? If this fails we presumably want to bail

Reply:
This always fails with `Error TPM deactivated from TPM_CreateEndorsementKeyPair` - I think this is expected. I've got this working in redshell, I'll know for sure once its working in Kokoro.